knative-operator/pkg/webhook/knativekafka/webhook_validating.go

```diff
@@ -109,7 +110,11 @@ func (v *Validator) validateShape(_ context.Context, ke *serverlessoperatorv1alp
 }
 
 // validate that KnativeEventing is installed as a hard dep
-func (v *Validator) validateDependencies(ctx context.Context, ke *serverlessoperatorv1alpha1.KnativeKafka) (bool, string, error) {
+func (v *Validator) validateDependencies(ctx context.Context, ke *serverlessoperatorv1alpha1.KnativeKafka, req admission.Request) (bool, string, error) {
+	// skip check if in deletion phase as Eventing maybe already deleted
+	if req.Operation == admissionv1.Delete {
```

Comment:
I wonder if this works as intended. We're already not specifying delete operations to be handled by the webhooks (see the CSV), so I think it's actually doing an update operation (because it has finalizers, so its only updating the deletionTimestamp).

As such, do we have to handle it that way, i.e. skip validations of `deletionTimestamp` is set?

Reply:
Yes supposed to have a finalizer removed when deletion timestamp is set ok. Let me update it.

Reply:
We might have to skip all validations not sure.

Reply:
you mean `validateLoneliness` for instance? yeah, IMO that check is not relevant for deletion.

Reply:
Ok I can refactor it in another PR.